### DISCUSSION TITLE
İbax Forum Bot!

### DISCUSSION THREAD

**Discussion by Majezikonline** [IBAX-io/go-ibax#457] [Ideas] [answered]
### Clear and concise description of the problem

İbax ön satışına katılan ve bot hesaplarla birden fazla işlem yapan kullanıcıların ön satıştan IP banı atılması

### Suggested solution

Ön satışa katılıp davet edilen her birey için,  ibax coinleri almadan önce referans olarak gelen bireylerin hesap doğrulamasının iyice araştırılıp; bot çıkan hesapların uzaklaştırılıp banlanması ve bot hesaplarï getiren kişilerin sistem tarafından kalıcı olarak banlanılması ve ibax coinlerine el konulup diğer üyelere dağıtılması

### Alternative

Bot hesaplarla referans kasan üyelerin sistemden komple banlanıp; verilen ibax coinlerin diğer kullanıcılara dağıtılması

### Validations

- [X] Read the [docs](https://docs.ibax.io).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by HalloKyle** [comment, marked as answer]
Thank you for your support, you can contact @ibaxtr via Telegram to become a mod

**Comment by impossible77777** [comment]
başarılar
